src/components/ConnectLogoHeader.tsx

```diff
@@ -37,24 +40,34 @@ export const ConnectLogoHeader: React.FC<ConnectLogoHeaderProps> = (props) => {
       </div>
     )
 
+  const aggregatorLogo = () =>
```

Comment:
Nit: Could we rename this to `defaultAggregatorLogo` to make it clearer that this returns the default aggregator logo?